### PR TITLE
#224 add a reference to this schema from the generated skeleton.xml

### PR DESCRIPTION
**Pull request by stepanov-dmitry**

### CODE REVIEW

**Comment by 0crat** [comment]
@paulodamaso[/z](https://www.0crat.com/u/paulodamaso) please, pay attention to this pull request

**Comment by 0crat** [comment]
Job [#339](https://github.com/yegor256/jpeek/issues/339) is now in scope, role is `REV`

**Comment by 0crat** [comment]
@paulodamaso[/z](https://www.0crat.com/u/paulodamaso) everybody who has role `REV` is banned at [#339](https://github.com/yegor256/jpeek/issues/339); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by stepanov-dmitry** [comment]
@0crat assign @GnusinPavel

**Comment by stepanov-dmitry** [comment]
assign @GnusinPavel

**Comment by 0crat** [comment]
> @0crat assign @GnusinPavel  ([here](https://github.com/yegor256/jpeek/issues/339#issuecomment-537134977))

@stepanov-dmitry You can't do "Request order start", unless you have one of these roles: `ARC`, `PO`; current roles of @stepanov-dmitry[/z](https://www.0crat.com/u/stepanov-dmitry) are: `DEV`

**Comment by stepanov-dmitry** [comment]
@paulodamaso, could you please assign this MR to someone for review?

File: src/test/java/org/jpeek/skeleton/SkeletonTest.java [hunk 1]
```diff
@@ -142,6 +142,20 @@ public void findFieldWithQualifiedName() {
         );
     }
 
+    @Test
+    public void findSchemaOfSkeleton() {
+        MatcherAssert.assertThat(
+                XhtmlMatchers.xhtml(
+                        new Skeleton(
+                                new FakeBase(
```
Comment (paulodamaso) [inline src/test/java/org/jpeek/skeleton/SkeletonTest.java]: @stepanov-dmitry please fix the indentation here

Comment (stepanov-dmitry) [reply src/test/java/org/jpeek/skeleton/SkeletonTest.java]: @paulodamaso thx, fixed. Do we have some code formatters for IDEs?

**Comment by stepanov-dmitry** [comment]
@paulodamaso, Do we have some code formatters for IDEs, applicable to this project?

**Comment by paulodamaso** [comment]
@stepanov-dmitry No, please take a look at [this article by @yegor256 ](https://www.yegor256.com/2018/01/16/educational-aspect-of-static-analysis.html) to see the reasons

**Comment by stepanov-dmitry** [comment]
@paulodamaso, I see, thank you! Could you please approve and merge it if you have no comments? Your comment has been applied.

**Comment by 0crat** [comment]
This pull request [#339](https://github.com/yegor256/jpeek/issues/339) is assigned to @jekeam[/z](https://www.0crat.com/u/jekeam), here is [why](https://www.0crat.com/footprint/C7JGJ00DP/1c3eb71b-e7c3-4b39-a16a-40eb5a7b8765); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by Jekeam** [comment]
@paulodamaso I see an error while checking, but I can't access how critical this is and understand why it will arise?
error: https://travis-ci.org/yegor256/jpeek/builds/592909160?utm_source=github_status&utm_medium=notification

**Comment by 0crat** [comment]
This pull request [#339](https://github.com/yegor256/jpeek/issues/339) is assigned to @mohamednizar[/z](https://www.0crat.com/u/mohamednizar), here is [why](https://www.0crat.com/footprint/C7JGJ00DP/7f1948cb-59fd-4a28-a19a-0a247a0a230a); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by mohamednizar** [comment]
@Jekeam , do we get this fail at all the time, I found some discussion related to this in Travis Forum, they are proposing add `dist: xenial` to the .travis.yml, how it will impact  to the project build ?

**Comment by mohamednizar** [comment]
> @paulodamaso I see an error while checking, but I can't access how critical this is and understand why it will arise?
> error: https://travis-ci.org/yegor256/jpeek/builds/592909160?utm_source=github_status&utm_medium=notification

Then we need to give a try on add `dist:trusty` to the .travis.yml , if don't find any issues with your code since we are working with openJDK8,  find this discussion on travis community  : https://travis-ci.community/t/error-installing-oraclejdk8-expected-feature-release-number-in-range-of-9-to-14-but-got-8/3766/6

**Comment by Jekeam** [comment]
> > @paulodamaso I see an error while checking, but I can't access how critical this is and understand why it will arise?
> > error: https://travis-ci.org/yegor256/jpeek/builds/592909160?utm_source=github_status&utm_medium=notification
> 
> Then we need to give a try on add `dist:trusty` to the .travis.yml , if don't find any issues with your code since we are working with openJDK8, find this discussion on travis community : https://travis-ci.community/t/error-installing-oraclejdk8-expected-feature-release-number-in-range-of-9-to-14-but-got-8/3766/6

Does this need to be done on a new ticket?

**Comment by mohamednizar** [comment]
No need there is an open issue regarding this, refer #338

@paulodamaso , #338 is the blocker for the future pull request. I ave created a pull request on the pull request #344  @Jekeam you can try rebuild as the  fix resolved the issue.

@paulodamaso the pull request can be accept after merge of #344

**Comment by paulodamaso** [comment]
@mohamednizar Right, thanks for the bug solution, I'll assign it to you so you got the points on merge

**Comment by stepanov-dmitry** [comment]
@mohamednizar Since #344 has been merged by @mohamednizar, can we merge this pull request as well? Or you have another comments?

**Comment by paulodamaso** [comment]
@stepanov-dmitry Build is failing, I think you should merge master branch

**Comment by 0crat** [comment]
@mohamednizar[/z](https://www.0crat.com/u/mohamednizar) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by mohamednizar** [comment]
@stepanov-dmitry can you merge the master and push to the pull request please.

**Comment by mohamednizar** [comment]
@0crat status

**Comment by stepanov-dmitry** [comment]
@mohamednizar, Sure, but I have failing CI build. Please give me more time to clarify a reason.

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/yegor256/jpeek/issues/339#issuecomment-544622773))

@mohamednizar This is what I know about this job in [C7JGJ00DP](https://www.0crat.com/p/C7JGJ00DP), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#339](https://github.com/yegor256/jpeek/issues/339) is [in scope](http://www.zerocracy.com/policy.html#14) for 21days
  * The role is `REV`
  * The job is assigned to @mohamednizar[/z](https://www.0crat.com/u/mohamednizar) for [18days](http://www.zerocracy.com/policy.html#8)
  * There is no monetary reward attached, it's a [free](http://www.zerocracy.com/policy.html#2) job
  * The job doesn't have any [impediments](http://www.zerocracy.com/policy.html#9)
  * The [budget](http://www.zerocracy.com/policy.html#4) is 15 minutes/points
  * These users are banned and won't be assigned:
    * @stepanov-dmitry[/z](https://www.0crat.com/u/stepanov-dmitry): This user reported the ticket
    * @jekeam[/z](https://www.0crat.com/u/jekeam): User was resigned from the ticket
  * Job [footprint](https://www.0crat.com/footprint/C7JGJ00DP?q=%7Bjob%3A%27gh%3Ayegor256%2Fjpeek%23339%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by stepanov-dmitry** [comment]
@mohamednizar, Sure, but I have failing CI build. Please give me more time to clarify a reason.

**Comment by mohamednizar** [comment]
@stepanov-dmitry  any updates ?

**Comment by stepanov-dmitry** [comment]
@mohamednizar, fixed, review please again and we will ready to merge.

**Comment by mohamednizar** [comment]
@paulodamaso we  can merge this pull request

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by 0crat** [comment]
@fabriciofx[/z](https://www.0crat.com/u/fabriciofx) please review this job completed by @mohamednizar[/z](https://www.0crat.com/u/mohamednizar), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
Code review was too long (29 days), architects (@paulodamaso) were penalized, see [§55](http://www.zerocracy.com/policy.html#55)

**Comment by 0crat** [comment]
The job [#339](https://github.com/yegor256/jpeek/issues/339) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Comment by fabriciofx** [comment]
@stepanov-dmitry please, fix the PR description including the #issue there.

**Comment by stepanov-dmitry** [comment]
@fabriciofx What do you mean? It's already there.
image

**Comment by stepanov-dmitry** [comment]
@fabriciofx, Done.

**Comment by fabriciofx** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @mohamednizar[/z](https://www.0crat.com/u/mohamednizar)

**Comment by 0crat** [comment]
Quality review completed: +4 point(s) just awarded to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx)

**Pull request merged**